app/models/manageiq/providers/amazon/cloud_manager/auth_key_pair.rb

```diff
@@ -15,16 +17,6 @@ def self.raw_create_key_pair(ext_management_system, create_options)
     raise MiqException::Error, err.to_s, err.backtrace
   end
 
-  def self.validate_create_key_pair(ext_management_system, _options = {})
-    if ext_management_system
-      {:available => true, :message => nil}
-    else
-      {:available => false,
-       :message   => _("The Keypair is not connected to an active %{table}") %
-         {:table => ui_lookup(:table => "ext_management_system")}}
-    end
-  end
-
```

Comment:
I've noticed similar validation put into `supports` blocks.
It feels like it is not needed here, but technically, the ems is not required for authentications of auth key pairs.

Do we want to:
- add the ems requirement to `AuthKeyPair` or `Authentication`
- add the validation back in?
- not worry about it

Reply:
@kbrock we should be asking the ems if it `supports?(:auth_key_pair_create)`  This set of PRs is focused on AuthKeyPair, so not going off on tangents for other types.